src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -25,6 +25,8 @@
 #include "../Satellites/SatPlanet.h"
 #include "SysPlanetA.h"
 
+#include "../icosahedron.h"
```

Comment:
Realistically, you've completely replaced this file.

It would be much easier to review if you named it something else, and deleted the original. That way github doesn't try to diff the two unrelated implementations.

Reply:
I'll just remove this file then. The commented out block isn't really important or final.